### PR TITLE
1777: Show timestamp for uploaded SR reports

### PR DESCRIPTION
**Pull request by chrismiceli**
add timestamp next to SR report for download

### CODE REVIEW

**Comment by chrismiceli** [comment]
I expect some feedback on this, but thought I'd start the conversation with a PR. Here is what this looks like:
image
@cpeel @srjfoo

File: project.php [hunk 1]
```diff
@@ -1763,17 +1763,29 @@ function echo_download_zip( $link_text, $discriminator )
...
     echo "\n";
 }
 
+function echo_last_modified($last_modified)
+{
+    global $datetime_format;
+
+    if (isset($last_modified))
+    {
+        echo " (", strftime($datetime_format, filemtime($file)), ")";
```
Comment (cpeel) [inline project.php]: `s/filemtime($file)/$last_modified/`

Comment (chrismiceli) [reply project.php]: This is what happens when I do a small refactor after testing but before pushing like a fool.

Comment (cpeel) [reply project.php]: I assumed as much - it happens, don't worry about it! 😁

Comment (srjfoo) [reply project.php]: Join the club. 😁

**Pull request merged**